src/screens/VaultScreen.js

```diff
@@ -43,33 +44,20 @@ export default class VaultScreen extends React.Component {
     };
     this.falseState = this.falseState.bind(this);
     this.setOpposite = this.setOpposite.bind(this);
-    this.renderScrollView = this.renderScrollView.bind(this);
+    this.renderResourceContent = this.renderResourceContent.bind(this);
   }
 
   async componentDidMount() {
-    await Font.loadAsync({
-      'source-sans-pro-black': require('../.././assets/fonts/SourceSansPro-Black.ttf'),
-      'source-sans-pro-black-italic': require('../.././assets/fonts/SourceSansPro-BlackItalic.ttf'),
-      'source-sans-pro-bold': require('../.././assets/fonts/SourceSansPro-Bold.ttf'),
-      'source-sans-pro-bold-italic': require('../.././assets/fonts/SourceSansPro-BoldItalic.ttf'),
-      'source-sans-pro-extra-light': require('../.././assets/fonts/SourceSansPro-ExtraLight.ttf'),
-      'source-sans-pro-extra-light-italic': require('../.././assets/fonts/SourceSansPro-ExtraLightItalic.ttf'),
-      'source-sans-pro-italic': require('../.././assets/fonts/SourceSansPro-Italic.ttf'),
-      'source-sans-pro-light': require('../.././assets/fonts/SourceSansPro-Light.ttf'),
-      'source-sans-pro-light-italic': require('../.././assets/fonts/SourceSansPro-LightItalic.ttf'),
-      'source-sans-pro-regular': require('../.././assets/fonts/SourceSansPro-Regular.ttf'),
-      'source-sans-pro-semibold': require('../.././assets/fonts/SourceSansPro-SemiBold.ttf'),
-      'source-sans-pro-semibold-italic': require('../.././assets/fonts/SourceSansPro-SemiBoldItalic.ttf'),
-    });
+    { fonts(); }
```

Comment:
I'm not sure what exactly is the error you're getting but you probably need an `await fonts()` since your `this.setState({fontLoaded: true});` needs to be executed **after** the font load has completed.